.changeset/ten-cougars-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Support markdown draft pages
```

Comment:
Could you give more a description here and give a code example? This will appear in our changelog.

Reply:
I have added more description, please check.